src/hb-ot-shape-complex-indic.cc

```diff
@@ -1472,6 +1472,8 @@ decompose_indic (const hb_ot_shape_normalize_context_t *c,
     case 0x0931u  : return false; /* DEVANAGARI LETTER RRA */
     case 0x0B94u  : return false; /* TAMIL LETTER AU */
 
```

Comment:
No need for the new-line here. Just sort these two with the previous two. Also add link to github issue.